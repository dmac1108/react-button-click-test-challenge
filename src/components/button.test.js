import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json'

import Button from './button';

describe('<Button />', () => {
    it('Should call props.onClick when the button is clicked', () => {
        const wrapper = shallow(<Button />);
        wrapper.find('button').simulate('click');
        expect(toJson(wrapper)).toMatchSnapshot()
    });
});
