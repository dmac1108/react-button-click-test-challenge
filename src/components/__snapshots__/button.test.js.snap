// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Should call props.onClick when the button is clicked 1`] = `
<button>
  Click me!
</button>
`;
